test_runner/regress/test_tenant_relocation.py

```diff
@@ -420,7 +420,7 @@ def test_tenant_relocation(
 
             # wait for tenant to finish attaching
             tenant_status = new_pageserver_http.tenant_status(tenant_id=tenant_id)
-            assert tenant_status["state"]["slug"] in ["Attaching", "Active"]
+            assert tenant_status["state"]["slug"] in ["Attaching", "Activating", "Active"]
```

Comment:
Quickly it sounds like this whole assertion means:

```
assert status not in ["Loading", "Broken", "Stopping"]
```

As in it exists to ensure attach has not yet failed, and to short circuit the 10s wait.

I think the whole stmt could be removed without any loss of functionality.

Reply:
you are right, and the next line will actually drive it to either Active or failure, removing it altogether